buildenv/docker/mkdocker.sh

```diff
@@ -223,10 +229,15 @@ fi
 }
 
 install_centos_packages() {
-  echo "RUN yum -y update \\"
-if [ $version = 6.9 ] ; then
-  echo " && yum -y install https://repo.ius.io/ius-release-el6.rpm \\"
+if [ $version = 6 ] ; then
+  echo "RUN sed -i -e 's|mirrorlist|#mirrorlist|' \\"
+  echo "           -e 's|#baseurl=http://mirror.centos.org/centos/\$releasever|baseurl=https://vault.centos.org/6.10|' \\"
+  echo "           /etc/yum.repos.d/CentOS-Base.repo"
+  echo ""
 fi
+  echo "RUN yum -y update \\"
+  echo " && echo "assumeyes=1" >> /etc/yum.conf \\"
```

Comment:
What is this for? Isn't it redundant with the `-y` option in `yum install -y`?

Reply:
The extra line for yum.conf was because I had a failure/prompt with the epel-release repo line.  But I cannot seem to reproduce in a minimal testcase.  I removed.